xs2a-adapter-service-impl/src/main/java/de/adorsys/xs2a/adapter/impl/http/wiremock/WiremockStubDifferenceDetectingInterceptor.java

```diff
@@ -116,7 +116,14 @@ private void analyzeRequestUrl(Request.Builder builder, String requestUrl) {
 
     @SuppressWarnings("unchecked")
     private Optional<String> getStubRequestUrl(Map<String, Object> jsonFile) {
-        return Optional.of((String) ((Map<String, Object>) jsonFile.get(REQUEST)).get("url"));
+        Map<String, Object> json = (Map<String, Object>) jsonFile.get(REQUEST);
+        if (json.containsKey("url")) {
+            return Optional.of((String) json.get("url"));
+        } else if (json.containsKey("urlPattern")) {
+            return Optional.of((String) json.get("urlPattern"));
+        } else {
```

Comment:
2nd else is not necessary

Reply:
Is it? This shouldn't be the case but what if the Map doesn't have `url` or `urlPattern`, e.g. typo - `urlPatern`?

Reply:
<img width="1625" alt="Screenshot 2020-09-15 at 10 13 07" src="https://user-images.githubusercontent.com/17544420/93177837-10ca1900-f73c-11ea-83eb-cef856e33cfe.png">

Reply:
I mean you can omit the last else statement and just return empty optional

Reply:
I see, adjusted